### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by devoncarew**
Add a badge for the travis build status.

### CODE REVIEW

**Comment by devoncarew** [comment]
@nex3 @kevmoo

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# yaml
```
Comment (kevmoo) [inline README.md]: Don't include the package name here – redundant!

Comment (devoncarew) [reply README.md]: It's a common practice for packages as far as I'm aware (https://github.com/dart-lang/dart-pad and others).

Comment (kevmoo) [reply README.md]: A common practice we should undo. It's completely redundant - see https://pub.dartlang.org/packages/angular_forms

Comment (kevmoo) [reply README.md]: *something something hobgoblins something something*

Comment (devoncarew) [reply README.md]: I down-vote your opinion, but will update this PR.

**Comment by devoncarew** [comment]
updated

**Pull request merged**